ansible/configs/kni-osp/default_vars.yml

```diff
@@ -0,0 +1,686 @@
+      port_range_max: 4789
+      remote_group: "master_sg"
+      description: "VXLAN from master"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 4789
+      port_range_max: 4789
+      remote_group: "worker_sg"
+      description: "VXLAN"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 6081
+      port_range_max: 6081
+      remote_group: "master_sg"
+      description: "Geneve from master"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 6081
+      port_range_max: 6081
+      remote_group: "worker_sg"
+      description: "Geneve"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "worker_sg"
+      description: "Worker ingress internal (tcp)"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "master_sg"
+      description: "Worker ingress from master (tcp)"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "master_sg"
+      description: "Worker ingress from master (udp)"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "worker_sg"
+      description: "Worker ingress internal (udp)"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 10250
+      port_range_max: 10250
+      remote_group: "master_sg"
+      description: "master ingress kubelet secure from master"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 10250
+      port_range_max: 10250
+      remote_group: "worker_sg"
+      description: "master ingress kubelet secure"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 30000
+      port_range_max: 32767
+      remote_group: "worker_sg"
+      description: "worker ingress services (tcp)"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 30000
+      port_range_max: 32767
+      remote_group: "worker_sg"
+      description: "worker ingress services (udp)"
+    - protocol: vrrp
+      direction: ingress
+      remote_ip_prefix: "{{ ocp_network_subnet_cidr }}"
+      description: "VRRP"
+
+################################################################################
+################################################################################
+### Common Host settings
+################################################################################
+################################################################################
+# Other Options are: file, satellite and rhn
+#If using repo_method: satellite, you must set these values as well.
+# satellite_url: satellite.example.com
+# satellite_org: Sat_org_name
+# satellite_activationkey: "rhel7basic"
+
+#repo_method: file
+#repo_version: "3.10"
+# Do you want to run a full yum update
+update_packages: false
+common_packages:
```

Comment:
this common_packages list would only work for rhel7. If you are using rhel8 and common role, it should be `common_packages_el8`